sqe-http-api/Helpers/SystemEmailer.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Threading.Tasks;
+using MailKit.Security;
+using Microsoft.AspNetCore.Identity.UI.Services;
+using MimeKit;
+using SmtpClient = MailKit.Net.Smtp.SmtpClient;
+
+namespace SQE.SqeHttpApi.Server.Helpers
+{
+    public class DevEmailSender : IEmailSender
+    {
+        /// <summary>
+        /// Sends an email
+        /// </summary>
+        /// <param name="email">Email address to send the message to.</param>
+        /// <param name="subject"></param>
+        /// <param name="htmlMessage"></param>
+        /// <returns></returns>
+        public async Task SendEmailAsync(string email, string subject, string htmlMessage)
```

Comment:
Do we want some code that takes Razor templates and turns them into emails? Or is this overkill?

Reply:
Its not a bad idea, especially if we will be sending several types of emails, then we could use a standardized template.  I would like to do this, but I will push it off for now.